src/AuthenticationInfo.js

```diff
@@ -26,7 +26,7 @@ export default class AuthenticationInfo {
     }
 
     /**
-     * Returns access_token without validity check.
+     * @returns access_token associated with LWA response
```

Comment:
LWA response is abstracted away for users of AuthenticationInfo. 
Just say provides access_token for accessing Alexa or something on those lines.